outport/factory/outportFactory.go

```diff
@@ -1,25 +1,14 @@
 package factory
 
 import (
-	logger "github.com/ElrondNetwork/elrond-go-logger"
-	"github.com/ElrondNetwork/elrond-go/core"
-	"github.com/ElrondNetwork/elrond-go/core/check"
-	"github.com/ElrondNetwork/elrond-go/data"
-	"github.com/ElrondNetwork/elrond-go/epochStart"
-	"github.com/ElrondNetwork/elrond-go/epochStart/notifier"
+	indexerFactory "github.com/ElrondNetwork/elastic-indexer-go/factory"
 	"github.com/ElrondNetwork/elrond-go/outport"
-	driversFactory "github.com/ElrondNetwork/elrond-go/outport/drivers/factory"
-	"github.com/ElrondNetwork/elrond-go/sharding"
 )
 
-var log = logger.GetOrCreate("outport/factory")
-
 // ArgsOutportFactory is structure used to store all components that are needed to create a new instance of
 // OurportHandler
 type ArgsOutportFactory struct {
-	ArgsElasticDriver  *driversFactory.ArgsElasticDriverFactory
-	EpochStartNotifier sharding.EpochStartEventNotifier
-	NodesCoordinator   sharding.NodesCoordinator
+	ArgsElasticDriver *indexerFactory.ArgsIndexerFactory
```

Comment:
do we still need the ArgsOutportFactory struct since it contains only the ArgsElasticDriver field?

Reply:
You right I can remove this extra structure, but in case of a new driver that maybe need some extra components we will need it again.

Reply:
we will create extra fields then. We should not make assumptions on the future code on the mid to long term basis

Reply:
removed this extra structure